reverse_engineering/databaseService/helpers/connection.js

```diff
@@ -86,10 +86,10 @@ class AzureActiveDirectoryMFAConnection extends Connection {
 			return axiosExtendedToken;
 		}
 
-		const msalToken = await this.#getTokenByMSAL();
-		if (msalToken) {
-			return msalToken;
-		}
+		// const msalToken = await this.#getTokenByMSAL();
```

Comment:
we can not use msal lib because it enforces the origin itself during the oauth flow and we actually don't need it to retrieve the token.  Axios should be replaced by @hackolade/fetch lib.